vmdb/app/models/miq_expression.rb

```diff
@@ -61,6 +63,12 @@ class MiqExpression
     cloud_tenants
     compliances
     compliance_details
+    configuration_profiles
+    configuration_managers
+    configured_systems
+    customization_scripts
+    customization_script_ptable
+    customization_script_medium
```

Comment:
but I'm confused...I thought this section just required table names, not necessarily association names.